editor/property_selector.cpp

```diff
@@ -135,9 +135,9 @@ void PropertySelector::_update_search() {
 			search_options->get_theme_icon(SNAME("Quaternion"), SNAME("EditorIcons")),
 			search_options->get_theme_icon(SNAME("AABB"), SNAME("EditorIcons")),
 			search_options->get_theme_icon(SNAME("Basis"), SNAME("EditorIcons")),
+			search_options->get_theme_icon(SNAME("NodePath"), SNAME("EditorIcons")),
 			search_options->get_theme_icon(SNAME("Transform3D"), SNAME("EditorIcons")),
 			search_options->get_theme_icon(SNAME("Color"), SNAME("EditorIcons")),
-			search_options->get_theme_icon(SNAME("NodePath"), SNAME("EditorIcons")),
```

Comment:
Why this move? The types should be kept in the order of `core/variant/variant.h`:
```
                // misc types
                COLOR,
                STRING_NAME,
                NODE_PATH,
                RID,
```

Not sure why `StringName` isn't included but the rest should be kept ordered.

Reply:
There was a problem with the way the icons appeared within the visual script based on their orderings here. In particular, Transform3D would should up where it should be path and vise versa, so by switching them this way they appeared as they should within the editor. Through trial and error, I found that having this arrangement seemed to resolve it. `StringName` was never there to begin with, not sure why it's missing 🤔

Reply:
There was a problem with the way the icons appeared within the visual script based on their orderings here. In particular, Transform3D would show up where it should be NodePath and vise versa, so by switching them this way they appeared as they should within the editor. Through trial and error, I found that having this arrangement seemed to resolve it. `StringName` was never there to begin with, not sure why it's missing 🤔

Reply:
Alright, I've tested re-ordering the types as they appear in `core/variant/variant.h` here's: what the Node3D looks like now:

<img width="169" alt="node3d" src="https://user-images.githubusercontent.com/62965063/126396963-98add846-e00c-4cfe-9a73-62696ba8251d.PNG">
This seems more accurate than before, though, I don't really know for sure.